viz/pkg/healthcheck/healthcheck.go

```diff
@@ -165,14 +156,78 @@ func (hc *HealthChecker) vizCategory() *healthcheck.Category {
 				}
 
 				// Check for relevant pods to be present
-				err = healthcheck.CheckForPods(pods, []string{"linkerd-grafana", "linkerd-prometheus", "linkerd-web", "linkerd-tap"})
+				err = healthcheck.CheckForPods(pods, []string{"linkerd-web", "linkerd-tap", "linkerd-metrics-api", "tap-injector"})
 				if err != nil {
 					return err
 				}
 
 				return healthcheck.CheckPodsRunning(pods, "")
 			}))
 
+	checkers = append(checkers,
+		*healthcheck.NewChecker("prometheus is installed and configured correctly").
+			WithHintAnchor("l5d-viz-prometheus").
+			Warning().
+			WithCheck(func(ctx context.Context) error {
+				// TODO: Skip if prometheus is disabled
```

Comment:
Is there more to do for this than I'm thinking? We can just check the values config and Prometheus is not enabled then it's skipped?

Reply:
Its harder I think as there is no values config in `linkerd-viz` 🤔

Reply:
Hm yea that does make it difficult.

Reply:
After some off GitHub discussion this will be covered in a follow-up early next week 👍